shipstation_integration/orders.py

```diff
@@ -60,7 +60,7 @@ def list_orders(
 		if not last_order_datetime:
 			# Get data for the last day, Shipstation API behaves oddly when it's a shorter period
 			last_order_datetime = datetime.datetime.utcnow() - datetime.timedelta(
-				hours=24
+				hours=72
```

Comment:
We had hard-coded this for our use, but I think it might be time to move this into its own field. I think the field could be inside the Shipstation Settings document itself, so every instance can handle its own schedule.

Reply:
agreed, for sure!